azure-pipelines.yml

```diff
@@ -13,23 +13,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+#
+# This file defines an Azure Pipeline build for testing Flink. It is intended to be used
+# with a free Azure Pipelines account.
+# It has the following features:
+#  - default builds for pushes / pull requests to a Flink fork and custom AZP account
```

Comment:
What parts of this file are related to `Flink fork and custom AZP account`?

Reply:
I clarified the language.